ui/components/MesheryAdapterPlayComponent.js

```diff
@@ -52,6 +53,11 @@ import fetchAvailableAddons from './graphql/queries/AddonsStatusQuery';
 import fetchAvailableNamespaces from "./graphql/queries/NamespaceQuery";
 import ReactSelectWrapper from "./ReactSelectWrapper";
 import MesheryMetrics from "./MesheryMetrics"
+import { errorHandlerGenerator, showProgress, successHandlerGenerator } from "./ConnectionWizard/helpers/common";
+import { pingKubernetes } from "./ConnectionWizard/helpers/kubernetesHelpers";
```

Comment:
instead of using the one from ConnectionWizard, you can just use the general one. it should be present in `/helpers/kubernetesHelpers;`

Reply:
Not able to find one.😅 Ok, I will add `handLeKuberntesClick` in this component.
What is the drawback of using one from`ConnectionWizard`??

Reply:
// @warunicorn19